index.js

```diff
@@ -0,0 +1,185 @@
+// INDEX
+// - Variables
+// - Initialize
+// - Navigation
+// - Step 1
+// - Step 2
+// - Step 3
+
+
+
+const step0 = document.getElementById('step0');
+const step1 = document.getElementById('step1');
+const step2 = document.getElementById('step2');
+const step3 = document.getElementById('step3');
+const tax = 0.08;
```

Comment:
Just a good-to-know: primitive constants are usually named in all caps. Primitive meaning simple values like numbers and strings. Eg. `const TAX = 0.08;`

The steps would stay as is because they're not totally constant, their live dom nodes that can and will mutate.

Reply:
That's a good differentiation for the steps, will do.